packages/crashlytics/ios/RNFBCrashlytics/RNFBCrashlyticsInitProvider.m

```diff
@@ -52,15 +52,6 @@ + (BOOL)isCrashlyticsCollectionEnabled {
 }
 
 + (void)configureWithApp:(FIRApp *)app {
-  if ([app isDefaultApp]) {
-    // TODO(salakar): 6.x/7.x Option to disable auto init
-    // TODO(salakar): If disabled; when the default app is initialized from JS land then init crashlytics (register a block handler somehow in RNFBApp?)
-    if ([[RNFBJSON shared] contains:KEY_CRASHLYTICS_DEBUG_ENABLED]) {
-      [Crashlytics sharedInstance].debugMode = [[RNFBJSON shared] getBooleanValue:KEY_CRASHLYTICS_DEBUG_ENABLED defaultValue:NO];
```

Comment:
It looks like the way to turn it on in debug now is `-FIRDebugEnabled` in iOS, in Android unsure? https://firebase.google.com/docs/crashlytics/test-implementation-new-sdk?platform=ios#enable_debug_logging